airflow/sensors/sql_sensor.py

```diff
@@ -28,9 +28,10 @@
 class SqlSensor(BaseSensorOperator):
     """
     Runs a sql statement repeatedly until a criteria is met. It will keep trying until
-    success or failure criteria are met, or if the first cell is in (0, '0', ''). Optional success
-    and failure callables are called with the first cell returned as the argument. If success
-    callable is defined the sensor will keep retrying until the criteria is met.
+    success or failure criteria are met, or if the first cell is in (0, '0', '', 'None').
+    An allow_null parameter exclude 'None' results from failure criteria.
```

Comment:
it's not failure, right? it's just "not done", i.e. poke return false and keep retrying.